austation/code/modules/mining/machine_vending.dm

```diff
@@ -0,0 +1,11 @@
+/obj/machinery/mineral/equipment_vendor
+	var/list/ignore_list = list(
+		"Minebot Upgrade: A.I.",
+		"Mining Bot Companion"
+	)
+
+/obj/machinery/mineral/equipment_vendor/Initialize()
+	..()
+	for(var/datum/data/mining_equipment/I in prize_list)
```

Comment:
is this meant to be ignore_list?

Reply:
var/I are itemes that may need to be <b>I</b>gnored , to be found in prize_list

Reply:
Oh I see what you did, checking the prize list for ignored. I expected the opposite for some reason, both work fine

Reply:
I've been totally misunderstanding what you meant and I'm sorry.
Will get on this tonight.